honeycomb.go

```diff
@@ -80,6 +80,15 @@ func getVendorOptionSetters() []launcher.Option {
 	opts := []launcher.Option{
 		WithHoneycomb(),
 	}
+	if endpoint := os.Getenv("HONEYCOMB_API_ENDPOINT"); endpoint != "" {
+		opts = append(opts, launcher.WithExporterEndpoint(endpoint))
+	}
+	if endpoint := os.Getenv("HONEYCOMB_TRACES_API_ENDPOINT"); endpoint != "" {
```

Comment:
Do these env vars take priority over the `OTEL_TRACES_EXPORTER` / `OTEL_METRICS_EXPORTER` endpoints?

Reply:
Yes, vendor options are applied after generic OTel ones.

Options are applied in order: base launcher options -> vendor options -> user options 👍🏻